### PR TITLE
feat(entropy): add worker deployment

### PR DESCRIPTION
**Pull request by FemiNoviaLina**
add `entropy worker` to entropy helm chart.

### CODE REVIEW

File: stable/entropy/values.yaml [hunk 1]
```diff
@@ -52,3 +52,47 @@ app:
     # NEW_RELIC_APP_NAME: entropy
 
   secretConfig: {}
+
+workers:
```
Comment (mabdh) [inline stable/entropy/values.yaml:56]: Let's also make it possible to put multiple worker deployments for this?

Comment (FemiNoviaLina) [reply stable/entropy/values.yaml:56]: this will require changes in the template, should we create a new template for the workers?

Comment (mabdh) [reply stable/entropy/values.yaml:56]: hmm you are correct, I think we can keep this as it is, user could have multiple helm release if they need.

but the `workers` name is plural. isn't it a bit misleading?

Comment (FemiNoviaLina) [reply stable/entropy/values.yaml:56]: oh right, will use `worker` instead

**Review by mabdh — APPROVED** [review]

**Pull request merged**